proposed/http-message.md

```diff
@@ -550,8 +550,7 @@ interface MessageInterface
     public function getHeader($name);
 
     /**
-     * Retrieves the line for a single header, with the header values as a
-     * comma-separated string.
+     * Retrieves a comma-concatenated string of the values for a single header.
```

Comment:
I've never heard the phrase "comma-concatenated" before. I know what you mean, but there may be a slightly clearer way of saying it?